src/dsp_tools/commands/xmlupload/models/ingest.py

```diff
@@ -94,7 +93,7 @@ def _ingest(self, filepath: Path) -> IngestResponse:
         Returns:
             IngestResponse: The internal filename of the uploaded file.
         """
-        filename = urllib.parse.quote(unicodedata.normalize("NFKD", filepath.name))
+        filename = urllib.parse.quote(filepath.name)
```

Comment:
@siers : Previously, I had tried out to do an NFKD normalization of files with special characters, before constructing the ingest URL. But now, I see that it also works without. What do you think is better / more future-proof: Doing it or not? 

(I'm refering to the POST /projects/<shortcode>/assets/ingest/filename route)

Reply:
Since it is now normalized in the backend, it is not strictly necessary.

Reply:
Thanks, then I won't do it.